payloads/external/iPXE/Makefile

```diff
@@ -80,10 +80,10 @@ endif
 build: config $(CONFIG_SCRIPT)
 ifeq ($(CONFIG_HAS_SCRIPT),y)
 	echo "    MAKE       $(project_name) $(TAG-y) EMBED=$(CONFIG_SCRIPT)"
-	$(MAKE) -C $(project_dir)/src $(IPXE_BUILD_TARGET) EMBED=$(CONFIG_SCRIPT) $(PXE_MAKE_OPTS)
+	$(MAKE) -C $(project_dir)/src $(IPXE_BUILD_TARGET) EMBED=$(CONFIG_SCRIPT) $(PXE_MAKE_OPTS) -j $(CPUS)
```

Comment:
Would it make sense to add this to `PXE_MAKE_OPTS` instead?

Reply:
yeah that makes more sense :) Fixed in dd0e7d42296f4ef0091af7d97a514f33006a68e8